.github/workflows/windows.yml

```diff
@@ -260,6 +268,13 @@ jobs:
 
       - name: Test
         run: |
+          if ("${{ matrix.enable-reference-count }}" -eq "yes") {
+            pwd
+            ls
+            ls "C:\Program Files\${{ matrix.label }}\"
+            Write-Output "loose-mroonga-enable-reference-count = ON" | `
+            Add-Content "C:\Program Files\${{ matrix.label }}\my.ini"
```

Comment:
We can just use `--mysqld=-loose-plugin-mroonga-enable-reference-count=ON` `mysql-test-run.pl` option: https://github.com/mroonga/mroonga/commit/f5fa385802784ca626530b6862ceb8da0a80188b